src/Nethermind/Nethermind.Consensus/IGasLimitCalculator.cs

```diff
@@ -7,6 +7,6 @@ namespace Nethermind.Consensus
 {
     public interface IGasLimitCalculator
     {
-        long GetGasLimit(BlockHeader parentHeader);
+        long GetGasLimit(BlockHeader parentHeader, long? desiredGasLimit = null);
```

Comment:
Something to think about - can we avoid changing this interface, I kind of don't think we should?